docs/core_docs/docs/integrations/vectorstores/weaviate.mdx

```diff
@@ -6,18 +6,21 @@ import CodeBlock from "@theme/CodeBlock";
 
 # Weaviate
 
-Weaviate is an open source vector database that stores both objects and vectors, allowing for combining vector search with structured filtering. LangChain connects to Weaviate via the `weaviate-ts-client` package, the official Typescript client for Weaviate.
+Weaviate is an open source vector database that stores both objects and vectors, allowing for combining vector search with structured filtering.
+LangChain connects to Weaviate via the `weaviate-ts-client` package, the official Typescript client for Weaviate.
 
 LangChain inserts vectors directly to Weaviate, and queries Weaviate for the nearest neighbors of a given vector, so that you can use all the LangChain Embeddings integrations with Weaviate.
 
 ## Setup
 
+Weaviate has their own standalone integration package with LangChain, accessible via [`@langchain/weaviate`](https://www.npmjs.com/package/@langchain/weaviate) on NPM!
+
 import IntegrationInstallTooltip from "@mdx_components/integration_install_tooltip.mdx";
 
 <IntegrationInstallTooltip></IntegrationInstallTooltip>
 
 ```bash npm2yarn
-npm install @langchain/openai weaviate-ts-client graphql @langchain/community
+npm install @langchain/weaviate @langchain/openai graphql @langchain/community
```

Comment:
We should not need all these deps anymore?

Reply:
ya just looked, correct on `graphql` but we still need openai, and community